docs/standard/security/how-to-use-data-protection.md

```diff
@@ -1,29 +1,35 @@
 ---
 title: "How to: Use Data Protection"
 description: Learn how to use data protection by accessing the data protection API (DPAPI) in .NET.
-ms.date: "03/30/2017"
+ms.date: 07/14/2020
 ms.technology: dotnet-standard
 dev_langs: 
   - "csharp"
   - "vb"
 helpviewer_keywords: 
   - "DPAPI"
-  - "encryption [.NET Framework], data protection API"
-  - "data [.NET Framework], decryption"
+  - "encryption [.NET], data protection API"
+  - "data [.NET], decryption"
   - "ProtectedMemory class, about ProtectedMemory class"
   - "ProtectedData class, about ProtectedData class"
-  - "cryptography [.NET Framework], data protection API"
-  - "data protection API [.NET Framework]"
+  - "cryptography [.NET], data protection API"
+  - "data protection API [.NET]"
   - "decryption"
-  - "data [.NET Framework], encryption"
+  - "data [.NET], encryption"
 ms.assetid: 606698b0-cb1a-42ca-beeb-0bea34205d20
 ---
 # How to: Use Data Protection
-The .NET Framework provides access to the data protection API (DPAPI), which allows you to encrypt data using information from the current user account or computer.  When you use the DPAPI, you alleviate the difficult problem of explicitly generating and storing a cryptographic key.  
+
+> [!NOTE]
+> This article applies to Windows.
+>
+> For information about ASP.NET Core, see [ASP.NET Core Data Protection](/aspnet/core/security/data-protection/introduction).
+
+.NET provides access to the data protection API (DPAPI), which allows you to encrypt data using information from the current user account or computer.  When you use the DPAPI, you alleviate the difficult problem of explicitly generating and storing a cryptographic key.  
   
- Use the <xref:System.Security.Cryptography.ProtectedMemory> class to encrypt an array of in-memory bytes.  This functionality is available in Microsoft Windows XP and later operating systems.  You can specify that memory encrypted by the current process can be decrypted by the current process only, by all processes, or from the same user context.  See the <xref:System.Security.Cryptography.MemoryProtectionScope> enumeration for a detailed description of <xref:System.Security.Cryptography.ProtectedMemory> options.  
+Use the <xref:System.Security.Cryptography.ProtectedMemory> class to encrypt an array of in-memory bytes.  This functionality is available in .NET Framework (not .NET Core or .NET 5) on Microsoft Windows XP and later operating systems.  You can specify that memory encrypted by the current process can be decrypted by the current process only, by all processes, or from the same user context.  See the <xref:System.Security.Cryptography.MemoryProtectionScope> enumeration for a detailed description of <xref:System.Security.Cryptography.ProtectedMemory> options.  
```

Comment:
I think we can avoid the OS version reference since everything supported is XP or later.

Since this type has been removed from .NET Core/.NET5+ maybe just cut the paragraph?